src/Microsoft.AspNetCore.Razor.Sdk/build/netstandard2.0/Razor.Sdk.CurrentVersion.targets

```diff
@@ -209,6 +217,21 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="ResolveRazorCompileInputs" DependsOnTargets="$(ResolveRazorCompileInputsDependsOn)">
   </Target>
 
+  <Target Name="ResolveRazorEmbeddedResources" Condition="'$(EmbedRazorGenerateSources)'=='true'">
```

Comment:
Moved this from code generation to SDK since it doesn't really require anything from code generation.